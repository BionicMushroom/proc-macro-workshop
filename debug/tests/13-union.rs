use derive_debug::CustomDebug;

#[derive(CustomDebug)]
union Test {
    field0: u32,
    field1: f32,
}

fn main() {}
