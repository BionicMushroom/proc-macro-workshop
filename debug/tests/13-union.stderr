error: expected struct or enum, found `union`
 --> tests/13-union.rs:4:1
  |
4 | union Test {
  | ^^^^^
